### PR TITLE
 currency exchange

### PR DESCRIPTION
**Pull request by tamana-Balkhi**
in this pull request:
no linters 👍 
use git flow 👍 
JavaScript best practice 👍 

 **project requirement** 
Home page

When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
Along with the category name, you will display some numeric values per category that come from the API.
When the user clicks (or taps) on a category item, the application navigates to the details page.
Details page
the webapp retrieves data from the API to show detailed data in the category.
When the user clicks on the "Back" button (<), the user navigates to the home page

Testing requirements

Create unit tests for pure functions in your Redux code.
Create integration tests for your application using the React Testing Library.
You may need to mock the access to the API, so that your tests don't send actual requests.
You may need to mock the connection to the Redux Store.
Technical requirements

The project is a single-page application (SPA) built with React and Redux.
The data retrieved from the API should be stored in the Redux store.
You should filter the data that you retrieve from the API using a Filter stateless component.
Every page (the main page and the pages for each item) should have a unique route within the SPA.
The project should be deployed and accessible online (add a link to the online version of your app to your README file).

### CODE REVIEW

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @tamana-Balkhi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title and description looks professional✔️
- Descriptive commit messages✔️
- No linter errors✔️
- You have a descriptive and professional looking readme file👍

## Required Changes ♻️
_Check the comments under the review._


- [ ] Kindly record a (5 minutes maximum) video as part of the project requirement, in the video presentation, highlight the following: (1) The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project should be made clear. (2) The video should contain a demo of the project, highlighting all the project features. (3) The video should contain a highlight about a piece of the code you find interesting or you are proud of making.

- [ ] kindly update the readme file so that you include the "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

- [ ] Kindly deploy the application using Heroku, Netlify or GitHub Pages. Make sure that the project is deployed properly, the home and detail page should work as expected. A link to the deployed project should be present in the README.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @tamana-Balkhi , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

gif

## Suggested changes
- [ ] Nice work done on your project, like the previous reviewer said kindly make sure your life demo link is working, your demo link is currently redirecting to your readme file,


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/home.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        <Nav text="Currencies Exchange">
+          <select onChange={showPreferedCurrency}>
+            {/* default options to be all currencies */}
+            <option value="allc">All Currencies</option>
+            {currenciesArray.map((currency) => (
+              <option key={currency.code} value={currency.code}>
+                {currency.name}
+              </option>
+            ))}
+          </select>
```
Comment (FrederickMih) [inline src/components/home.js:61]: - [Optional] In my opinion, we could have just displayed all currencies on the homepage without the select options. Next, we could have used ES6 JS filter() method to filter out currencies by name using a search bar in a stateless Filter component. This way, it'll be more easier for the user to search for a particular item(currency) by just introducing the name in the search bar.



Below is a screenshot of how the dropdown options occur on mobile screens. Not just responsive. Kindly make sure to fix the issue.



Screenshot from 2022-10-13 03-41-48

**Review by FrederickMih — APPROVED** [review]
Hi @tamana-Balkhi, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## To Highlights :green_circle: 

- Correct Gitflow has been followed :heavy_check_mark: 
- PR is descriptive :heavy_check_mark: 


## Optional changes 

-  There is still some deployment failing checks. Kindly make sure to fix them.
-  Your app still contains boilerplates from CRA. Some unused images, logos, and files are found in your root directory and public folder. Delete them to render your codebase clean.

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**